internal/oci/oci_linux.go

```diff
@@ -162,3 +176,35 @@ func metricsToCtrStats(c *Container, m *cgroups.Metrics) *ContainerStats {
 		PIDs:        pids,
 	}
 }
+
+// getTotalInactiveFile returns the value if `total_inactive_file` as integer
+// from `/sys/fs/cgroup/memory/memory.stat`. It returns an error if the file is
+// not parsable.
+func getTotalInactiveFile() (uint64, error) {
+	const memoryStat = "/sys/fs/cgroup/memory/memory.stat"
```

Comment:
not sure how we can deal with it in cgroup v2, as this information is not present in the root cgroup and we should probably lookup every child.  For now could we just add a check and return 0 if it is under cgroup v2?

Reply:
Yes sure, I will add a check for cgroups v2.